### PR TITLE
Update tests to use 3.2.0 version of artifacts 

### PR DESCRIPTION
**Pull request by onobc**
The SCDF server core tests are broken as they rely on the `1.0.0.BUILD-SNAPSHOT` version of the stream apps which no longer exist in Artifactory.

Moves to the latest `3.2.0` artifacts and adjusts tests accordingly.

### CODE REVIEW

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AppRegistryControllerTests.java [hunk 1]
```diff
@@ -368,15 +368,15 @@ public void testListSingleApplication() throws Exception {
 		mockMvc.perform(get("/apps/source/time").accept(MediaType.APPLICATION_JSON)).andDo(print())
 				.andExpect(status().isOk()).andExpect(jsonPath("name", is("time")))
 				.andExpect(jsonPath("type", is("source")))
-				.andExpect(jsonPath("$.options[*]", hasSize(6)));
+				.andExpect(jsonPath("$.options[*]", hasSize(1)));
```
Comment (onobc) [inline spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AppRegistryControllerTests.java:371]: This was simplified in the recent move to the SCDF sample apps repo.

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java [hunk 2]
```diff
@@ -860,8 +860,7 @@ public void testStreamWithShortformProperties() throws Exception {
 		assertThat(logSpec.getApplicationProperties().get("level")).isNull();
 
 		SpringCloudDeployerApplicationSpec timeSpec = parseSpec(timePackage.getConfigValues().getRaw());
-		assertThat(timeSpec.getApplicationProperties().get("trigger.fixed-delay")).isEqualTo("2");
-		assertThat(timeSpec.getApplicationProperties().get("fixed-delay")).isNull();
+		assertThat(timeSpec.getApplicationProperties().get("fixed-delay")).isEqualTo("2");
```
[the remark this answers is not in the document]
Comment (jvalkeal) [reply spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java:863]: Maybe something changed in metadata. It was anyway based on very old apps.

**Review by jvalkeal — APPROVED** [review]

**Comment by jvalkeal** [comment]
It's fun to fix those asserts when small things change.

**Comment by onobc** [comment]
> It's fun to fix those asserts when small things change.

Ahh yes... it is the nature of the test pyramid and the further we get away from "unit tests" the more likely other things will make us break. But valuable tests nonetheless. 

Ty for review @jvalkeal

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (onobc) [reply spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java:863]: Will keep an eye on this if we see any issues arising out of it but should be good.

**Comment by onobc** [comment]
Me and @dturanski co-reviewed this code via Zoom today. Going to merge now.

**Pull request merged**